code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm

```diff
@@ -318,10 +318,10 @@
 	..()
 	// Make sure only one official queen exists at any point.
 	if(!alien_queen_exists(1,H))
-		H.real_name = "alien queen ([alien_number])"
+		H.real_name = "Queen ([alien_number])"
```

Comment:
Alien queen sounds far better than just Queen in game, to be fair.

Reply:
I think the standard "alien queen (1)" looks pretty bad by itself. Might look better if I capitalized both instead of just removing "alien", actually.